packages/tanstack-react-start/package.json

```diff
@@ -74,13 +74,13 @@
     "tslib": "catalog:repo"
   },
   "devDependencies": {
-    "@tanstack/react-router": "1.115.2",
-    "@tanstack/react-start": "1.115.2",
+    "@tanstack/react-router": "^1.121.0",
```

Comment:
noting that we're going from an explicit version to a range here

Reply:
Noted. TanStack Start is still in beta and receives frequent updates. Just to stay updated and maintain stability